code/modules/mob/dead/observer/observer.dm

```diff
@@ -240,9 +240,11 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
  * Increase the brightness of a color by calculating the average distance between the R, G and B values,
  * and maximum brightness, then adding 30% of that average to R, G and B.
  *
- * I'll make this proc global and move it to its own file in a future update. |- Ricotez
+ * I'll make this proc global and move it to its own file in a future update. |- Ricotez - UPDATE: They never did :(
```

Comment:
do it fool

Reply:
> do it fool

I could, but that would require moving it around to somewhere else, which is out of scope for this PR.